gpu/octree/src/cuda/octree_host.cu

```diff
@@ -174,7 +173,7 @@ void pcl::device::OctreeImpl::radiusSearchHost(const PointType& query, float rad
             int beg = host_octree.begs[node_idx];
             int end = host_octree.ends[node_idx];
 
-            end = beg + min<int>((int)out.size() + end - beg, max_nn) - (int)out.size();
+            end = beg + std::min<int>((int)out.size() + end - beg, max_nn) - (int)out.size();
```

Comment:
Please check this. Adding `std` in cuda file (where it has been removed before)

Reply:
Comparing with `abe501a`, I removed `using namespace std;`, so I had to qualify this to get it compile with `clang`.